src/routes/addresses/index.ts

```diff
@@ -131,13 +135,24 @@ async function addresses(fastify: FastifyInstance) {
 
         if (rows[0].amount) {
           for (const asset of rows[0].amount) {
-            const registryData = await fetchAssetMetadata(asset.unit);
+            const unit = `${asset.policy_id}${asset.asset_name}`;
```

Comment:
`asset.unit` is no longer available?

Reply:
That is correct - we can keep it in the underlying SQL query but still need both name and policy for the validation function. It has to be concatenated in either SQL or JS, I chose the latter.